pom.xml

```diff
@@ -110,22 +69,6 @@
                 <scope>provided</scope>
             </dependency>
 
-            <!-- Guava -->
```

Comment:
We'll still need this. Guava 15 won't work for some of our projects, which is what wcm-io is bringing in.

Reply:
Done, I added the Guava 22